packages/celest/README.md

```diff
@@ -1,3 +1,9 @@
-## Celest
+## celest
 
-The Flutter cloud platform.
+> **Note**: Celest is currently in private beta, but you can still [download](https://celest.dev/download) the CLI and experience all of Celest in your local environment. To stay up-to-date on the future of Celest, including getting access to managed deployments, join our waitlist at [celest.dev](https://celest.dev).
+
+Celest is the Flutter cloud platform, enabling Flutter and Dart developers to define their backend infrastructure purely in Dart.
+
+This package contains our "Cloud Widget" primitives which you compose to define your backend. Most of the time, they do not need to be directly instantiated and can be inferred by the Celest CLI. For example, Celest Functions are just top-level Dart functions!
```

Comment:
This package contains our "Cloud Widget" primitives which you compose to define your backend. For example, Celest Functions are just top-level Dart functions!

I've removed the sentence in the middle. LMK